### DISCUSSION TITLE
Some components don't implement FromReflect, eg PointLight

### DISCUSSION THREAD

**Discussion by raffaeleragni** [bevyengine/bevy#8774] [Q&A] [answered]
Hi, I found out that some bevy components don't implement FromReflect, even though I believe they should be part of a saveable/serializable context. Either for an editor, or for network synchronization. In my specific case is for network synchronization.
The one component that for now I found doesn't implement FromReflect is `PointLight`. I suspect now that there may be more that I am not aware of and will encounter them as I go.

On a side note, what about assets passed across network? Anybody did any asset sync crate? Such as Mesh, Material and all the stuff you can have a Handle<> on. If not I'll have to build that one myself too I guess.

**Comment by alice-i-cecile** [comment, marked as answer]
I agree: these should all impl FromReflect. Can you open an issue or PR please? :)

**Comment by raffaeleragni** [comment]
https://github.com/bevyengine/bevy/pull/8776
